group-membership/README.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Scale access to external vendors and partners
+description: Share documents, events, and communications with users outside 
+your domain
+labels:  Sheets, Groups for Business, Apps Script
+material_icon: group_work
+create_time: 2019-08-30
+update_time: 2019-12-27
+---
+Contributed by Tech and Eco, follow me on
+[Twitter](https://twitter.com/TechandEco)!
+
+Use a Google Group to work cross-functionally with vendors, partners,
+customers, and volunteers outside of your domain, and scale access
+to work assignments and leverage security controls. You can add an external
+user's Google email (this can be their own G Suite account, a G Suite user
+account you created for them within your domain, or a free Gmail account) to
+your G Suite Google group (not available for consumer Google Groups) via a
+Google Sheet to manage users in bulk and keep track of the onboarding history.
+
+Top 6 benefits of this solution:
+
+ 1. Adds a new external user to a G Suite Google Group you create
+   (example: externalteam@mydomain.com) once, and it allows users to open all
+   resources that are shared with that group moving forward (calendar, files,
+   training site, dashboards (built on DataStudio), etc.
+
+ 1. When adding one or more users to your Google Group, users can receive a
+    welcome email with the links they can access. Creating a Google Site is
+    optional but recommended to centralize information in a
+    beautiful website interface.
+
+ 1. The template of the welcome email is composed in a Google Doc. You
+    can use seperate Google Docs for different recipients to customize
+    messaging.
+
+ 1. Capture the email addresses of users in your Sheet, and add them in bulk to
+    the desired Google Groups at a later time by changing the column "Allowed"
+    to "yes" for each row of users.
```

Comment:
Can you make the capitalization match for the Sheet template and code? In the demo Sheet should that column show a data validation drop down for yes and no?

Reply:
Made the change in the sheet to have 'yes' be in lowercase. And no it does not include a validation drop down.

Reply:
Can you add the data validation to the sheet? It is "safer" and seems like a best practice for this use case. We have it for a similar use case in:
https://github.com/gsuitedevs/solutions/tree/master/vacation-requests

Reply:
Good call, added data validation.